axiom/tests/test_span_event.c

```diff
@@ -256,6 +256,47 @@ static void test_span_event_category(void) {
   tlib_pass_if_str_equal("Category should be the one we set - http", "http",
                          nr_span_event_get_category(event));
 
+  nr_span_event_set_category(event, NR_SPAN_MESSAGE);
+  tlib_pass_if_str_equal("Category should be the one we set - message",
+                         "message", nr_span_event_get_category(event));
+
+  nr_span_event_destroy(&event);
+}
+
+static void test_span_event_spankind(void) {
+  nr_span_event_t* event = nr_span_event_create();
+
+  // Test : the default is NULL (spankind must be explicitly set)
+  tlib_pass_if_str_equal("The default category", NULL,
+                         nr_span_event_get_spankind(event));
+
+  // Test : A null event returns NULL
```

Comment:
Instead of comments, let's make them test messages!

Reply:
Please see: https://github.com/newrelic/newrelic-php-agent/pull/991/commits/862604b081029206c2072275588076691d30560a